Quick Start Documentation/FPGA Addon Quick Start Guide.md

```diff
@@ -0,0 +1,282 @@
+
+<style>
+    h1{
+        color:#87dfd6;
+    }
+    h2{
+        color:#01a9b4;
+    }
+</style>
```

Comment:
Remove this and all other tags (<br> and the like). I would like to stick to vanilla markdown, for this documentation.

Reply:
And for the <br> tags, I believe they are redundant: 

![image](https://user-images.githubusercontent.com/61490253/86105909-97ba0e80-bac8-11ea-8b64-f12f406e6c47.png)

Reply:
I agree with removing the <style> tag - actually, I did not know that it is banned for md on github ... however the rest of the tags (<br>,) are actually whitelisted and they can be properly rendered. If it is ok, I would like to ask to keep them, since they help with the overall esthetics. 

https://gist.github.com/kivikakk/622b5dcf395e26c49e2334f0eb19e6f9